UnityProject/Assets/Scripts/Learning/UI/ProtipUI.cs

```diff
@@ -36,14 +36,14 @@ public void ShowTip(string tip, float showDuration = 25f, Sprite img = null, Spr
 			StartCoroutine(TipShowCooldown(showDuration));
 			switch (animation)
 			{
-				case SpriteAnimation.NONE:
-					break;
 				case SpriteAnimation.ROCKING:
 					StartCoroutine(DoImageRockAnimations());
 					break;
 				case SpriteAnimation.UPDOWN:
 					StartCoroutine(DoImageJumpAnimations());
 					break;
+				default:
+					break;
```

Comment:
Log.Error the unexpected value you received. It has helped me before to find issues where I wouldn't think was one

Reply:
I think this should replace `NONE` so a log is not needed